include/config.h

```diff
@@ -81,7 +81,7 @@ struct mako_config {
 	struct mako_style superstyle;
 
 	struct {
-		enum mako_button_binding left, right, middle;
+		enum mako_button_binding left, right, middle, touch;
```

Comment:
Can we move the new `touch` member out of the `button_bindings` struct? It's not a button binding.

Reply:
no worries, it's still considered a type of mako_button_binding ?

Reply:
What about renaming `mako_button_binding` to `mako_binding` for consistency?

Reply:
sure